web/themes/custom/votegov/src/sass/uswds-overrides/usa-site-alert.scss

```diff
@@ -66,26 +66,3 @@
 .usa-alert__text{
   margin-inline-start: 2.5rem;
 }
-
```

Comment:
moved this code back to to the `usa-alert` file and adding it back in since it was removed